Dockerfile74

```diff
@@ -0,0 +1,24 @@
+FROM php:7.4-cli
```

Comment:
Kde se používá 7.4? IMHO by to mělo testovat na těch verzích kde to běží + latest s allow fail

Reply:
nova queue bezi na 7.4. a tohle bude potreba i tam